java/examples/src/main/java/glide/examples/clients/GlideClient.java

```diff
@@ -0,0 +1,28 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.examples.clients;
+
+import glide.api.RedisClient;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.configuration.RedisClientConfiguration;
+import glide.examples.ExamplesApp;
+import java.util.concurrent.ExecutionException;
+
+/** Connect to Jedis client. See: https://github.com/redis/jedis */
+public class GlideClient {
```

Comment:
If our example app needs an ease-of-use wrapper around our code, it means that our code isn't easy to use, and should be refactored.
This shouldn't be here. If you think the existing APIs aren't good enough to use, we should spend more time refining the existing APIs. Having this here is a badge of shame on our APIs.